paths/entities/items/descriptions/singleLang.json

```diff
@@ -11,5 +11,26 @@
             "404": { "$ref": "../../../../responses.json#/NotFound" },
             "default": { "$ref": "../../../../responses.json#/UnexpectedError" }
         }
+    },
+    "put": {
+        "tags": ["items"],
+        "summary": "Add / Replace a description to an entity by language",
+        "parameters": [
+            { "$ref": "../../../../parameters.json#/entityId" },
+            { "$ref": "../../../../parameters.json#/lang" }
+        ],
+        "requestBody": { "$ref": "../../../../requests.json#/Item" },
```

Comment:
In this case, The request will be a Fingerprint (where the language key is not required):
```json
{
  "language": "en",
  "value": "Some description" 
}
```